### PR TITLE
Added first skeleton to fetch events from API server

### PR DESCRIPTION
**Pull request by botherder**

### CODE REVIEW

File: blockade/libs/events.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    """Client to interface with events for blockade."""
+
+    def __init__(self, *args, **kwargs):
+        """Setup the primary client instance."""
+        super(EventsClient, self).__init__(*args, **kwargs)
+
+    def get_events(self):
+        to_send = {'limit': 50}
+        response = self._send_data('GET', 'admin', 'get-events', to_send)
+        output = {'message': "Fetched events"}
```
Comment (9b) [inline blockade/libs/events.py]: This is just going to return the string, no actual events. Did you mean to return the actual response from the API back to the user?

Comment (botherder) [inline blockade/libs/events.py]: I was just being lazy for the moment as the results are anyhow displayed in the response output. I thought duplicating that would have been a bit redundant.

Comment (botherder) [inline blockade/libs/events.py]: How about using prettytables or something similar to produce the output?

Comment (9b) [inline blockade/libs/events.py]: We could do that. Thoughts on using `tabulate`?

Comment (botherder) [inline blockade/libs/events.py]: Fine for me.

**Pull request merged**